app/Services/Finance/PirepFinanceService.php

```diff
@@ -299,6 +299,15 @@ public function payExpensesForPirep(Pirep $pirep): void
                 $transaction_group = "Expense: {$expense->name}";
             }
 
+            if (filled($pirep->aircraft->subfleet->ground_handling_multiplier)
+                && $expense->multiplier
+                && $expense->multiplier == 1) {
```

Comment:
`===`, and why `=== 1` ?

Reply:
This is something i am using for a month or so, I really do not remember why I did double check there. Most probably only $expense->multiplier did not worked as i expected thus i had to use two checks. Or just me being weird things.